internal/gmtcli/info.go

```diff
@@ -116,13 +121,34 @@ func getRadioInfo(r gomesh.Radio, json bool) error {
 func showModemOptions(c *cli.Context) error {
 	fmt.Println("Modem Options")
 	printDoubleDivider()
-	fmt.Printf("'lf' for %s\n", gomeshproto.Config_LoRaConfig_LONG_FAST.String())
-	fmt.Printf("'vls' for %s\n", gomeshproto.Config_LoRaConfig_VERY_LONG_SLOW.String())
-	fmt.Printf("'ms' for %s\n", gomeshproto.Config_LoRaConfig_MEDIUM_SLOW.String())
-	fmt.Printf("'mf' for %s\n", gomeshproto.Config_LoRaConfig_MEDIUM_FAST.String())
-	fmt.Printf("'sl' for %s\n", gomeshproto.Config_LoRaConfig_SHORT_SLOW.String())
-	fmt.Printf("'sf' for %s\n", gomeshproto.Config_LoRaConfig_SHORT_FAST.String())
-	fmt.Printf("'lm' for %s\n", gomeshproto.Config_LoRaConfig_LONG_MODERATE.String())
+	fmt.Printf(
```

Comment:
Same formatting notes here, if we don't need to break up the lines like this can we return things to how they were before?